app/controllers/events_controller.rb

```diff
@@ -0,0 +1,65 @@
+class EventsController < ApplicationController
+  before_action :authenticate_user!, except: %i[index show]
+
+  def index
+    @events = Event.all
```

Comment:
- [ ] The events index page should have both a list of past and future events, that means you should pass at least two instance variables from the Events#index, @future_events and @past_events.
  - Don't make the logic to query the Events model at the view, that logic should be made in the Event model with scopes and those scopes called at the Events controller and assigned to the variables mentioned above.
  Read more on Rails scopes here: https://www.rubyguides.com/2019/10/scopes-in-ruby-on-rails/